formatting-requirements.txt

```diff
@@ -0,0 +1,6 @@
+# Dependencies for code formatting
+
+# black>=22.1,<23
```

Comment:
Black is out of beta, so we can rely on their stability policy

Reply:
I would extend comment here and reference a corresponding issue which references that the entire library should be blacked once more with an ignore for the diff.

Reply:
I added a comment, but we don't need to open an issue, since dependabot will remind us about this update :)